contracts/TracerPerpetualSwaps.sol

```diff
@@ -564,10 +570,12 @@ contract TracerPerpetualSwaps is ITracerPerpetualSwaps, Ownable {
     }
 
     function setFeeRate(uint256 _feeRate) external override onlyOwner {
+        require(_feeRate <= MAX_PERCENT, "TCR: Fee rate > 100%");
         feeRate = _feeRate;
     }
 
     function setMaxLeverage(uint256 _maxLeverage) external override onlyOwner {
```

Comment:
Should there also be a check for max leverage being required to be greater than (or equal to) 1 in WAD?

Reply:
Likewise with lowest max leverage

Reply:
It's not theoretically impossible for it to be less than 1.

Reply:
When would you want that? That sounds like overcollateralised accounts

Reply:
Since they should theoretically hold (fee will be valid as long as it's not negative, same with leverage), should these just be left to our current trust model where the Owner decides and accepts the risk?

Reply:
I'm not sure tbh, see what Patchy's decision is